docs/content/en/docs/reference/clusterspec/irsa.md

```diff
@@ -0,0 +1,117 @@
+
+The steps below are based on the [guide for configuring IRSA for DIY Kubernetes](https://github.com/aws/amazon-eks-pod-identity-webhook/blob/master/SELF_HOSTED_SETUP.md), with modifications specific to EKS Anywhere's cluster provisioning workflow. The main modification is the process of generating the keys.json document. As per the original guide, the user has to create the service account signing keys, and then use that to create the keys.json document prior to cluster creation. This order is reversed for EKS Anywhere clusters, so you will create the cluster first, and then retrieve the service account signing key generated by the cluster, and use it to create the keys.json document. The sections below show how to do this in detail.
+
+#### Create an OIDC provider and make its discovery document publicly accessible
+
+1. Create an s3 bucket to host the public signing keys and OIDC discovery document for your cluster as per [this section](https://github.com/aws/amazon-eks-pod-identity-webhook/blob/master/SELF_HOSTED_SETUP.md#create-an-s3-bucket). Ensure you follow all the steps and save the `$HOSTNAME` and `$ISSUER_HOSTPATH`.
+
+1. Create the OIDC discovery document as follows:
+
+```bash
+cat <<EOF > discovery.json
+{
+    "issuer": "https://$ISSUER_HOSTPATH",
+    "jwks_uri": "https://$ISSUER_HOSTPATH/keys.json",
+    "authorization_endpoint": "urn:kubernetes:programmatic_authorization",
+    "response_types_supported": [
+        "id_token"
+    ],
+    "subject_types_supported": [
+        "public"
+    ],
+    "id_token_signing_alg_values_supported": [
+        "RS256"
+    ],
+    "claims_supported": [
+        "sub",
+        "iss"
+    ]
+}
+EOF
+```
+
+1. Upload it to the publicly accessible S3 bucket:
+```bash
+aws s3 cp --acl public-read ./discovery.json s3://$S3_BUCKET/.well-known/openid-configuration
+```
+
+1. Create an [OIDC provider](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc.html) for your cluster. Set the `Provider URL` to `https://$ISSUER_HOSTPATH`, and audience to `sts.amazonaws.com`.
+
+1. Note down the `Provider` field of OIDC provider after it is created.
+
+1. Assign an IAM role to this OIDC provider. 
+
+	1. To do so from the AWS console, select and click on the OIDC provider, and click on **Assign role** at the top right.
+	2. Select **Create a new role**.
+	3. In the **Select type of trusted entity** section, choose Web identity.
+	4. In the Choose a web identity provider section:
+		For Identity provider, choose the auto selected Identity Provider URL for your cluster.
```

Comment:
```suggestion
      * For Identity provider, choose the auto selected Identity Provider URL for your cluster.
      * For Audience, choose sts.amazonaws.com.
```
I think you are looking for something like this. I think you can indent the nested numbered list 3 spaces, then indent the two bullets in another two spaces. You may have to play with it a bit to get it to work.